Pendulum/Data/Pendulum.xcdatamodeld/Pendulum.xcdatamodel/contents

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="21512" systemVersion="21G217" minimumToolsVersion="Automatic" sourceLanguage="Swift" usedWithCloudKit="YES" userDefinedModelVersionIdentifier="">
+    <entity name="CDEvent" representedClassName="CDEvent" syncable="YES" codeGenerationType="class">
+        <attribute name="date" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="id" optional="YES" attributeType="UUID" usesScalarValueType="NO"/>
```

Comment:
Double check if you really want these to be optional on create

Reply:
I originally had them as non-optional, but I preferred to set the defaults once (in Swift) than twice (in Swift *and* in the Model GUI), because they're always Optionals in Swift so we have to unwrap them to something anyway.

Reply:
The risk is that there's a small chance something falls through the cracks and you have multiple penpals without an ID or something in your state

Reply:
True. But what do you set a default ID to?!